README.md

```diff
@@ -1436,38 +1514,41 @@ along with these options:
 var view = Mikado.new(template, { reuse: false, pool: "key" });
 ```
 
-This will switch Mikado into a "explicit keyed" mode (shared).
+This will switch Mikado into an "explicit keyed" mode (shared).
 
 <a name="cross-shared"></a>
+
 #### 4. Cross-Shared (Hybrid)
 
-> The cross shared mode is a hybrid and takes the performance benefits of both shared pools and provides you an enhanced pooling of reusable components. This mode provide high performance as well as low memory allocation during runtime.
+> The cross shared mode is a hybrid and takes the performance benefits of both shared pools and provides you an enhanced pooling of reusable components. This mode provides high performance as well as low memory allocation during runtime.
+
+Add the attribute **_key_** to the **_root element_** of a template:
 
-Add the attribute ___key___ to the ___root element___ of a template:
 ```html
 <div key="data.id">
-    <div>User:</div>
-    <div>{{data.name}}</div>
+  <div>User:</div>
+  <div>{{data.name}}</div>
 </div>
 ```
 
 along with these options:
 
 ```js
-var view = Mikado.new(template, { pool: true } );
+var view = Mikado.new(template, { pool: true });
 ```
 
-This will switch Mikado into a "cross-shared-keyed" mode.
+This will switch Mikado into an "cross-shared-keyed" mode.
```

Comment:
That's wrong. The word "an" is only used if the following _sound_ is a vowel. This is clearly not the case here.

Probably a copy-and-paste mistake from "into an exclusive mode" further below.

Reply:
@ts-thomas Could you also review https://github.com/nextapps-de/mikado/issues/29#issuecomment-561680545 and I will fix both issues.

Reply:
Done.

Reply:
> When data is changed, the input elements will automatically update, as well as other turn around, when the input elements get new data the store will automatically update.

What does this mean "as well as other turn around" ?

Reply:
> When data is changed, the input elements will automatically update, as well as other turn around, when the input elements get new data the store will automatically update.

What does this mean "as well as other turn around" ?

Reply:
Sorry, just remove some words :)

> When data is changed, the input elements will automatically update, as well as ~~other turn around,~~ when the input elements get new data the store will automatically update.